### ISSUE TITLE
Fix insecure downloading of raspberrypi.org signing key

### ISSUE THREAD

**Issue by jturner314** [debian-pi/raspbian-ua-netinst#64]
#62 provides verified installation for http://archive.raspbian.org packages, but the installer script also installs the http://archive.raspberrypi.org repository. In order to do so, it downloads the http://archive.raspberrypi.org signing key insecurely with `wget` and adds it to the trusted Apt keyring without any authenticity checks. This download is vulnerable to attack. The best solutions that I can think of are:
- Include a known-good copy of the key in the installer image so that it doesn't have to be downloaded during installation.
- Don't add the http://archive.raspberrypi.org repository during unattended installation, and let users know in the README that they should add it manually if desired and verify the key themselves. This repository doesn't have many packages outside of the http://archive.raspbian.org repository, anyway (list below).

```
8086tiny
8086tiny-dev
compoundpi-client
compoundpi-common
compoundpi-docs
compoundpi-server
icelib
icelib-dev
omxplayer
oracle-java7-jdk
oracle-java8-jdk
penguinspuzzle
pistore
pypy-setuptools
pypy-smbus-cffi
pypy-upstream
pypy-upstream-dev
pypy-upstream-doc
python3-lirc
python3-picamera
python3-pifacecad
python3-pifacecad-emulator
python3-pifacecommon
python3-pifacedigital-emulator
python3-pifacedigitalio
python3-pifacedigital-scratch-handler
python3-rpi.gpio
python3-snap-camera
python-lirc
python-picamera
python-picamera-docs
python-pifacecad
python-pifacecommon
python-pifacedigitalio
python-rpi.gpio
raspberrypi-artwork
raspberrypi-bootloader
raspi-config
raspi-copies-and-fills
rpi-update
simbamond
smartsim
sonic-pi
wolfram-engine
xserver-xorg-video-fbturbo
```

**Comment by diederikdehaas** [comment]
The key used to sign the raspberrypi.org archive is now published securely on https://www.raspberrypi.org/raspberrypi.gpg.key
And ShitfPlusOne send me the following msg, just like plugwash has done earlier:

-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA1

I confirm the key being used to sign the raspberrypi.org repository is
pub   2048R/7FA3303E 2012-06-17
Key fingerprint = CF8A 1AF5 02A2 AA2D 763B  AE7E 82B1 2992 7FA3 303E
-----BEGIN PGP SIGNATURE-----
Version: GnuPG v1

iQEcBAEBAgAGBQJVSO8TAAoJEKQcgohUhKUlTT0H/3lc1X/p3DwgbSrdIiVNLawX
0dbkLrvvJCrOYaV/xztosDnTPU3AF3HFqjVWmliBMcW0rFBXWz03tfXD38OszEK7
jzt3oNdoVetCl5K8QCDHqMtwIgIMKg0w0oojFPwFAqxJ4i8QlfC6YwZtW0Wge9Zz
SwTWnZKbv8gvLRukNjHjGtddDKwX6yqg58McXkZ7VAdUFxujJ7iOnexAXXMz0b1N
Gzmec/QbE4LuhVKxTmH2howOGeXJ8UDCqDQDzbxbcQnTrk5wnVPrWMuVhSFH8BD+
ue5Nmc34zV98t2rheUkZZXpIjIcdzYrrakI2p8cWhu8CUivgndkOVCev/PaU7D4=
=G/So
-----END PGP SIGNATURE-----

**Comment by diederikdehaas** [comment]
For safe-keeping I've now posted both confirmations on paste.debian.net
plugwash: https://paste.debian.net/171594/
ShiftPlusOne: https://paste.debian.net/171595/

**Comment by diederikdehaas** [comment]
This issue is now fixed with the release of v1.0.7 of the Raspbian unattended netinstaller.
